spec/domain/queries/find_aggregations_spec.rb

```diff
@@ -0,0 +1,83 @@
+RSpec.describe Queries::FindAggregations do
+  it 'A hash with a Key with each metric name' do
```

Comment:
I feel this test title has implementation details - maybe something like `has aggregations for all metric names`?